Update README with instructions what to use instead of this action (#10)
Yleisradio/gh-project-actions: README.md

# gh-project-actions

**DEPRECATED**

> Github has very similar Action in their Marketplace, built by Github. 

Reviewer: ```suggestion
GitHub has a very similar Action in their Marketplace, built by GitHub. 
```

<https://github.com/marketplace/actions/add-to-github-projects>

> Functionally only difference is you can not specify the target column.
> However the project board can have a default column for new issues and PR's.

Reviewer: ```suggestion
Functionally the only difference is you cannot specify the target column.
However the project board can have a default column for new issues and PRs.
```

<https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#configure-built-in-automation>

--

> Github actions for automatic management of Github project

Reviewer: ```suggestion
GitHub actions for automatic management of GitHub project
```
